profiler/profiler.go

```diff
@@ -151,6 +151,10 @@ type Config struct {
 	// When true, collecting the goroutine profiles is disabled.
 	NoGoroutineProfiling bool
 
+	// When true, the agent sends all telemetries via OpenCensus exporter.
```

Comment:
It may be helpful to mention that these telemetries are visible in Cloud Trace and Cloud Monitoring. 

... via OpenCensus exporter, which can be viewed in Cloud Trace and Cloud Monitoring.

Reply:
Thank you. Followed your suggestion.